<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>managerView</class>
 <widget class="QMainWindow" name="managerView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="logout">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>410</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color:  rgb(255,0,0);
    border-radius: 3px;
	color: black;
}
QPushButton:pressed {
    background-color: rgb(203, 218, 249);
	color: black;
}</string>
    </property>
    <property name="text">
     <string>LOGOUT</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="itemtable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>871</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:lightyellow</string>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>106</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>20</number>
    </attribute>
    <column>
     <property name="text">
      <string>Codice</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Type</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Brand</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Processor</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Model</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Display (&quot;)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string> Price ($)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Memory (Gb)</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="ricerca" native="true">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>631</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>60</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Euro</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="codice_edit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="prezzo_edit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>98</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>.SF NS Text</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Ricerca per</string>
     </property>
    </widget>
    <widget class="QComboBox" name="itemType">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Qualsiasi</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Notebook</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Smartphone</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Smartwatch</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>20</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Tipo:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="search">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>80</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color:  rgb(255,0,0);
    border-radius: 3px;
	color: black;
}
QPushButton:pressed {
    background-color: rgb(203, 218, 249);
	color: black;
}</string>
     </property>
     <property name="text">
      <string>CERCA</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="brand_edit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>138</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Codice</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>PrezzoMin</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Brand</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addItem">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>80</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color:  rgb(255,0,0);
    border-radius: 3px;
	color: black;
}
QPushButton:pressed {
    background-color: rgb(203, 218, 249);
	color: black;
}</string>
     </property>
     <property name="text">
      <string>ADD ITEM</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="logged">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>410</y>
      <width>181</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Logged as:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
